src/renderers/testing/ReactTestRenderer.js

```diff
@@ -134,9 +136,9 @@ ReactComponentEnvironment.injection.injectEnvironment({
   replaceNodeWithMarkup: function() {},
 });
 
+
 var ReactTestRenderer = {
   create: ReactTestMount.render,
-
   /* eslint-disable camelcase */
```

Comment:
And keep this one, so the diff is smaller. In general better not to formatting changes to code you didn’t directly touch unless it’s horribly formatted